jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/FileID.java

```diff
@@ -255,6 +270,17 @@ public static boolean isHidden(Path base, Path path)
         return false;
     }
 
+    /**
+     * Is the URI pointing to a Java Archive (JAR) File (not directory)
+     *
+     * @param uri the uri to test.
+     * @return True if a .jar file.
+     */
+    public static boolean isJavaArchive(URI uri)
+    {
+        return ".jar".equals(getExtension(uri));
```

Comment:
Don't you want to use `.equalsIgnoreCase()`?

Reply:
`getExtension()` returns it in lowercase already.